server/config/config.go

```diff
@@ -223,6 +227,7 @@ func DefaultConfig() *Config {
 			IAVLCacheSize:       781250,
 			IAVLDisableFastNode: false,
 			AppDBBackend:        "",
+			MaxEventSize:        1000000, // 1MB
```

Comment:
We decided on 1MB basing off the attacker's event size (2.3 MB). This can be modified as we see fit, although 1 MB feels on the higher end of what we should have.